test/jdk/TEST.groups

```diff
@@ -635,13 +638,17 @@ jdk_core_manual_no_input_security = \
     sun/security/smartcardio/TestMultiplePresent.java \
     sun/security/smartcardio/TestPresent.java \
     sun/security/smartcardio/TestTransmit.java \
-    sun/security/tools/jarsigner/compatibility/Compatibility.java \
-    java/security/Policy/Root/Root.java
+    sun/security/tools/jarsigner/compatibility/Compatibility.java
 
-jdk_core_manual_requires_human_input = \
+jdk_core_manual_human = \
```

Comment:
jdk_core_manual_requires_human_input gave a good indication of what is needed with this test group ... maybe jdk_core_manual_interactive if we don't want to use requires_human_input

Reply:
The name "jdk_core_manual_requires_human_input" may not be suitable because some tests in this group require environment preparation or root privilege. I prefer the name "jdk_core_manual_interactive".